src/Goerli-DssSpell.t.sol

```diff
@@ -104,6 +66,22 @@ contract DssSpellTest is GoerliDssSpellTestBase {
     //     assertEq(join.bar(), 2.85 * RAY / 100);
     // }
 
+    function testCollateralIntegrations() private { // make public to use
```

Comment:
Hmm what's the point to leave the tests but without the example content?

Reply:
I've added test templates and moved the chainlog housekeeping into a separate test in 6694a70.